### PR TITLE
docs: :memo: major overhaul of iteration guide to reorganize it

### LINKED ISSUE #134: Update iteration guide on focusing more on the retrospective and less on the "what we did"
So we have more time to think about any struggles or challenges we had faced personally during the iteration, how we felt our own personal progress and contribution was and whether we felt we achieved what we individually wanted to do.

### PR DESCRIPTION
**Pull request by lwjohnst86**
## Description

Revised the iteration guide to be more organized and not just a single long list.

## Related Issues

Closes #134

### CODE REVIEW

File: how-we-work/iteration.qmd [hunk 1]
```diff
@@ -31,98 +31,150 @@ research and academic environments don't foster nor support nor are
...
+## End of iteration
+
+::: callout-important
+We track our iterations' progress and tasks on GitHub with [this project
+board](https://github.com/orgs/seedcase-project/projects/18/).
+:::
+
+The iteration's end is a last meeting for a debrief and retrospective to
+discuss how things went. These are between 1-2 hours and are preferably
+in person. It will have been scheduled during the planning meeting.
```
Comment (signekb) [inline how-we-work/iteration.qmd:149]:
```suggestion
in person. The time and place of this meeting will have been scheduled during the planning meeting.
```

File: how-we-work/iteration.qmd [hunk 2]
```diff
@@ -31,98 +31,150 @@ research and academic environments don't foster nor support nor are
...
+At the start of the iteration is the first planning meeting that is
+scheduled well in advance, for between 1-2 hours and (preferably) right
+after the previous iteration's debrief and retrospective. Preferably
+this meeting is in person. The aim of the meeting to discuss and plan
+the specifics of the iteration.
+
+::: callout-note
+A basic agenda for the meeting might be:
+
+-   Review and agree on the iteration aim and end date
```
Comment (signekb) [inline how-we-work/iteration.qmd:72]:
```suggestion
-   Assign timekeeper.
-   Review and agree on the iteration aim and end date
```

File: how-we-work/iteration.qmd [hunk 3]
```diff
@@ -31,98 +31,150 @@ research and academic environments don't foster nor support nor are
...
+-   Review the list of tasks already listed
+-   Brainstorm and add any other issues as needed
+-   Assign a team member to each task
+-   Schedule the next iteration's end and start meetings
+:::
```
Comment (signekb) [inline how-we-work/iteration.qmd:77]:
```suggestion
-   Review and agree on the iteration aim and end date.
-   Review the list of tasks already listed.
-   Brainstorm and add any other issues as needed.
-   Assign a team member to each task.
-   Schedule the next iteration's end and start meetings.
:::
```
Add `.` to fit other lists in this doc

File: how-we-work/iteration.qmd (hunk 2 above)
Comment (signekb) [reply how-we-work/iteration.qmd:72]: With `.` to fit other lists in this doc

File: how-we-work/iteration.qmd [hunk 4]
```diff
@@ -31,98 +31,150 @@ research and academic environments don't foster nor support nor are
...
+-   Evaluate the iteration aim.
+-   Go over what we accomplished and how we felt we did (including
+    struggles).
+-   Do a retrospective.
+:::
+
+During the debrief and retrospective meeting, we will:
+
+-   Assign a timekeeper to keep us on track and not spending too long on
+    any one topic.
```
Comment (signekb) [inline how-we-work/iteration.qmd:165]:
```suggestion
-   Assign a timekeeper to keep us on track and so we avoid spending too long on
    any one topic.
```

File: how-we-work/iteration.qmd [hunk 5]
```diff
@@ -31,98 +31,150 @@ research and academic environments don't foster nor support nor are
...
+-   Show off and (optionally) demo the output all together as a team.
```
Comment (signekb) [inline how-we-work/iteration.qmd:166]:
```suggestion
-   Show off and (optionally) demo the output of the iteration all together as a team.
```

File: how-we-work/iteration.qmd [hunk 6]
```diff
@@ -31,98 +31,150 @@ research and academic environments don't foster nor support nor are
...
+    give some feedback.
+-   Evaluate the aim and review whether we've achieved the aim.
+-   Briefly go over what we each did during the iteration and how we
+    felt we did. Include any barriers, struggles, or challenges that
+    came up during the iteration here.
+-   Do a "retrospective", where we individually write down how things
+    went ("keep doing" category) and what could be improved or removed
+    ("stop doing/improve on" category) in the process. The retrospective
+    should ideally be done in a collaborative "notetaking/post-it notes"
+    system ([Figma](https://www.figma.com/) is really nice for this).
```
Comment (signekb) [inline how-we-work/iteration.qmd:180]:
```suggestion
    system ([Figma](https://www.figma.com/) is really nice for this). These notes will be collected into topics and we will add action items to each of the topics. The team lead will add these action items to issues or posts on how we work.
```

**Review by signekb — COMMENTED** [review]
Very nice! The reorganisation is very nice :)

File: how-we-work/iteration.qmd [hunk 7]
```diff
@@ -31,98 +31,150 @@ research and academic environments don't foster nor support nor are
...
+An iteration starts with a first planning meeting that is
+scheduled well in advance, for between 1-2 hours and (preferably) right
+after the previous iteration's debrief and retrospective. Preferably
+this meeting is in person. The aim of the meeting to discuss and plan
+the specifics of the iteration.
+
+::: callout-note
+A basic agenda for the meeting might be:
+
+-   Review and agree on the iteration aim and end date.
```
Comment (lwjohnst86) [inline how-we-work/iteration.qmd:72]:
```suggestion
-   Assign a timekeeper.
-   Review and agree on the iteration aim and end date.
```

**Review by signekb — DISMISSED** [review]
Did some minor fixes, but looks good 👍

File: how-we-work/iteration.qmd [hunk 8]
```diff
@@ -31,98 +31,161 @@ research and academic environments don't foster nor support nor are
...
+-   Assign a team member to each task.
+-   Schedule the next iteration's end and start meetings.
+:::
+
+::: panel-tabset
+### Before meeting
+
+In general, only the team lead is required to prepare for the planning
+meeting. Before the planning meeting, the team lead will move any
+existing tasks (GitHub issues) into the iteration that is relevant to
```
Comment (lwjohnst86) [inline how-we-work/iteration.qmd:85]:
```suggestion
existing tasks (GitHub Issues) into the iteration that is relevant to
```

@signekb small thing, when issues are used with the word GitHub, they are capitalized, in line with GitHub's own documentation ("GitHub Issues" vs "create a new issue")

Comment (signekb) [reply how-we-work/iteration.qmd]: Ah, you're right!

**Pull request merged**